### ISSUE TITLE
Black version is no longer specified anywhere.

### ISSUE THREAD

**Issue by shaleh** [MichaelAquilina/flake8-spellcheck#76]
The switch to poetry dropped the locking to a specific version of black. The target_version in pyproject.toml says "py37".

**Comment by shaleh** [comment]
Ok, i just found it in the .pre-commit-config.yaml. That was not very obvious.

Is there a reason not to declare it somewhere more prominent and have pre-commit use that definition? If people want to configure their editor for flake8, isort, black, etc etc. support how will they know what versions or options are needed?

**Comment by djmattyg007** [comment]
I agree, it should be possible to run code quality tools without first trying to create a commit. I'd rather deal with issues earlier rather than later.

**Comment by MichaelAquilina** [comment]
Using `pre-commit` to ensure correct linting is pretty standard nowadays in most python projects (even [Django](https://github.com/django/django)).

I dont have any opposition to also putting black, isort and flake8 inside dev-requirements as I would assume this helps with the developer experience. The downside here is that we need to make sure the two declared versions are in sync (one in pyproject.toml and one in the pre-commit configuration).

However within teams I've been working on, maintaining and making sure these two values are in sync has not proven very troublesome so I think we can go down that route.
